### DISCUSSION TITLE
✅  Android version?

### DISCUSSION THREAD

**Discussion by JoKerIsCraZy** [isontheline/pro.webssh.net#551] [Q&A]
Hey i would like to ask why there is no android version of Webssh

**Comment by isontheline** [comment]
Hey @JoKerIsCraZy👋

That's because :
* When launching WebSSH back to 2012 there were already **good** SSH clients on Android (such as [JuiceSSH](https://juicessh.com/changelog#v1.0.4))
* There are now plenty (too many?) SSH apps on both Apple and Google stores and many of them are better than WebSSH
* I don't have time to maintain an application on both operating systems
